src/js/components/CompleteYourProfile/CompleteYourProfile2024.jsx

```diff
@@ -93,10 +93,6 @@ class CompleteYourProfile2024 extends Component {
     } else {
       this.setItemNotComplete(stepIdSignInToSave);
     }
-    this.setState({
```

Comment:
It makes sense to remove the `this.sortSteps()` portion of this setState, but we need to still update the state of howItWorksWatched and personalizedScoreIntroCompleted.